<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти налаштування</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkServer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>531</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввімкнути трансляцію веб сторінки з додатковою інформацією при переході на заблоковані сайти?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Налаштування</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPass2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cboLanguage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Українська</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deutsch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Русский</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cboTimeLocation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ukraine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>United_States</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Germany</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mosow</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtPass1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Мова графічного інтерфейсу:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Місцезнаходження. Використовується для доступу до часу через інтернет.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPass1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть старий пароль:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkWebBlock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>531</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввімкнути блокування веб-сайтів?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSoftBlock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>531</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввімкнути блокування програмного забезпечення?</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkChangePass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>531</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Змінити пароль?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPass2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть новий пароль:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
